packaging/installer/UPDATE.md

```diff
@@ -20,28 +21,35 @@ icon in the local Agent dashboard's top navigation. This modal informs you wheth
 
 ## Determine which installation method you used
 
+Starting with netdata v1.33.0, you can use Netdata itself to determine the installation type by running:
+
+```bash
+netdata -W buildinfo | grep 'Install type:'
+```
+
+If this produces no output, you have an older install and will have to manually look in the Netdata config directory.
 If you are not sure where your Netdata config directory is, see the [configuration doc](/docs/configure/nodes.md). In
 most installations, this is `/etc/netdata`.
 
 Use `cd` to navigate to the Netdata config directory, then use `ls -a` to look for a file called `.install-type`.
 
 -   If the `.install-type` file doex not exist, look for a file in the same directory called `.environment`.
-    -   If the `.environment` file does not exist, you probably installed Netdata using your system package manager
-        and should update it the same way you would run updates on the system itself.
-    -   If the `.environment` file does exist, then our [regular update method](#updates-for-most-systems) should
-        work correctly.
+    -   If the `.environment` file does not exist, then you have a ‘custom’ install.
+    -   If the `.environment` file does exist, then you have either a ‘static’ or ‘build’ install.
 -   If the `.install-type` file does exist, check it’s contents with `cat .install-type`.
-    -   If the `INSTALL_TYPE` key has a value of `custom`, you probably installed Netdata using your system
-        package manager and should update it the same way you would run updates on the system itself.
-    -   If the `INSTALL_TYPE` key has a value of `oci`, the install is from a Docker image.
-    -   Otherwise, the install should work with our [regular update method](#updates-for-most-systems).
+    -   The value of the `INSTALL_TYPE` key indicates what type of install you have.
 
-Next, use the appropriate method to update the Netdata Agent:
+The exact update method to use depends on the install type:
 
--   [Updates for most systems](#updates-for-most-systems)
--   [Docker](#docker)
--   [macOS](#macos)
--   [Manual installation from Git](#manual-installation-from-git)
+-   Installs with an install type of 'custom' usually indicate installing a third-party package through the system
```

Comment:
Another common case is building from source

Reply:
This was written with the (not unreasonable) assumption that anybody building from source by hand (the only way they should see a `custom` install type other than a third-party package) knows what they are doing well enough that they should not need this document.

Reply:
@Ferroin it could be not by hand, but just using kickstart.sh.

Reply:
The kickstart script never produces a `custom` install unless things are horribly broken. Any install done with it reports as either `kickstart-build` (for newer stuff) or `legacy-build` (for really old stuff that hasn’t been updated since before the addition of the install-type file).